src/components/form/Form.js

```diff
@@ -0,0 +1,204 @@
+
+    // ____________________________close popup
+    const closePopup = () => {
+        props.closePopup();
+    }
+
+    // ___________________________________ Google form data
+    const formName = 'entry.1259736291';
+
+    const formEmail = 'entry.1503900032';
+
+    const formRequest = 'entry.1763891937';
+
+    const formURL = 'https://docs.google.com/forms/d/18EfQ-4I_6IffAlGivd0wxGbebGZ1REUGuVuJe4_YvgE/formResponse';
+
+    let submittedForm = false;
+    let hiddenFrameLoaded = false;
+
+    function handleHiddenFrameLoaded() {
+        if (hiddenFrameLoaded) {
+            submittedForm = true;
+        } else {
+            hiddenFrameLoaded = true;
+        }
+    }
+
+    return (
+        <Overlay open={props.open}>
+            <Wrapper>
+                <Exit
+                    src={exit}
+                    onClick={closePopup} />
+                <Title>
+                    Leave your contacts and request,
+                    and our manager will reach out to you to
+                    establish our futher parntership!
+                </Title>
+                <Iframe
+                    title='hidden_iframe'
+                    name='hidden_iframe'
+                    id='hidden_iframe'
+                    onload={handleHiddenFrameLoaded} />
+                <ConnectForm
+                    ref={formRef}
+                    onChange={checkFormValidity}
+                    target='hidden_iframe'
+                    action={formURL}
+                    method='POST'
+                    onSubmit={closePopup}>
+                    <InputWrap>
+                        <Label
+                            $valid={nameValid}>
+                            Personal or company name *
+                            {!nameValid && errorMessage}
+                        </Label>
+                        <Input
+                            $valid={nameValid}
+                            type='text'
+                            name={formName}
+                            id='name'
+                            required
+                            onChange={handleChange}
+                            onBlur={updateErrors}
+                            value={inputs.name} />
+                    </InputWrap>
+                    <InputWrap>
+                        <Label
+                            $valid={emailValid}>
+                            Email *
+                            {!emailValid && errorMessage}
+                        </Label>
+                        <Input
+                            $valid={emailValid}
+                            type='text'
+                            id='email'
+                            name={formEmail}
+                            required
+                            onChange={handleChange}
+                            onBlur={updateErrors}
+                            value={inputs.email} />
+                    </InputWrap>
+                    <InputWrap>
```

Comment:
this field should be required to. I don't think it makes sense to be able to send the form with just your name and email with no message.

Reply:
What if instead of making it a required field, we put it in a stock text like "I would like to learn more about partnering with Awesome Container Company!"
If they decide to rewrite their own message, fine, but the field is already handled.

Reply:
I think that's fine to add, but they could erase the message and just send blank inquiries. Imagine how annoying it would be to receive 100 requests that are just emails/names with no indication of what they want.